specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4573,14 +4573,38 @@
           "type": "string",
           "description": "The namespace of the resource provider."
         },
+        "authorizations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceProviderAuthorization"
+          },
+          "description": "The authorizations fields of the resource provider."
+        },
+        "metadata": {
```

Comment:
What's this metadata? Does it have any required property?

Reply:
It's the metadata for the Resource Provider, if any. No required properties.